index.ts

```diff
@@ -311,10 +314,256 @@ export const User = {
+//       const { name: repo } = self.$argsAt(root.users.one.repos.one);
+
+//       await ensureWebhook(owner, repo, "release");
+//     },
+//     async unsubscribe(_, { self }) {
+//       const { name: owner } = self.$argsAt(root.users.one);
+//       const { name: repo } = self.$argsAt(root.users.one.repos.one);
+
+//       await removeWebhook(owner, repo, "release");
+//     },
+//   },
+//   pushed: {
+//     async subscribe(_, { self }) {
+//       const { name: owner } = self.$argsAt(root.users.one);
+//       const { name: repo } = self.$argsAt(root.users.one.repos.one);
+
+//       await ensureWebhook(owner, repo, "push");
+//     },
+//     async unsubscribe(_, { self }) {
+//       const { name: owner } = self.$argsAt(root.users.one);
+//       const { name: repo } = self.$argsAt(root.users.one.repos.one);
+
+//       await removeWebhook(owner, repo, "push");
+//     },
+//   },
+//   branches: () => ({}),
+//   commits: () => ({}),
+//   issues: () => ({}),
+//   pull_requests: () => ({}),
+//   releases: () => ({}),
+//   content: () => ({}),
+//   async license(_, { self }) {
+//     const { name: owner } = self.$argsAt(root.users.one);
+//     const { name: repo } = self.$argsAt(root.users.one.repos.one);
+//     const res = await client().licenses.getForRepo({ owner, repo });
+//     return res.data;
+//   },
+// };
+
+// export const OrgRepositoryCollection = {
+//   async one(args, { self, info }) {
+//     const { name: repo } = args;
+//     const { name: owner } = self.$argsAt(root.users.one);
+//     if (
+//       !shouldFetch(info, [
+//         "name",
+//         "repos",
+//         "issues",
+//         "pull_requests",
+//         "releases",
+//       ])
+//     ) {
+//       return { name: repo };
+//     }
+//     const result = await client().repos.get({ owner, repo });
+//     return result.data;
+//   },
+//   async page(args, { self }) {
+//     const { name: username } = self.$argsAt(root.users.one);
+
+//     const apiArgs = toGithubArgs({ ...args, username });
+//     const res = await client().repos.listForUser(apiArgs);
+
+//     return {
+//       items: res.data,
+//       next: getPageRefs(self.page(args), res).next,
+//     };
+//   },
+//   async search(args, { self }) {
+//     const { name: username } = self.$argsAt(root.users.one);
+//     const q = (args.q ?? "") + ` user:${username}`;
+
+//     const apiArgs = toGithubArgs({ ...args, q, username });
+//     const res = await client().search.repos(apiArgs);
+
+//     return {
+//       items: res.data.items,
+//       next: getPageRefs(self.search(args), res).next,
+//     };
+//   },
+// };
+
+
+
+const getOwner = (obj) => {
+  //get name of owner. this handles case where owner is user and case where owner is org
+  const { name: owner } = obj.$argsAt(root.users.one);
+  if (!owner){
+    const { name: owner } = obj.$argsAt(root.organizations.one);
+    return owner;
+  }
+  return owner;
+}
```

Comment:
Nice! If you want to make this a bit more concise, you could do something like:
```suggestion
// Handles case where owner is user and case where owner is org
const getOwner = (gref) => {
  const { name: owner } = gref.$argsAt(root.users.one) ?? gref.$argsAt(root.organizations.one);
  return owner;
}
```